vars/commonPipelineEnvironment.groovy

```diff
@@ -7,6 +7,10 @@ class commonPipelineEnvironment implements Serializable {
     //stores the gitCommitId as well as additional git information for the build during pipeline run
     String gitCommitId
     String gitSshUrl
+    String gitHttpsUrl
+    String gitFolder
```

Comment:
I have problems to understand what a `gitFolder` is. All other new variables are understandable for users with average git understanding.`gitfolder` is blurry.

Reply:
Maybe you have a better suggestion? If only doing it for github I would name it githubOrg. This does not really fit for git and there to my best knowledge we have also folders in the url.